transform/replacer_test.go

```diff
@@ -64,7 +70,7 @@ func TestReplacer_Transform(t *testing.T) {
 			src:      []byte("abcdefgabca"),
 			atEOF:    false,
 			nDst:     10,
-			nSrc:     10,
+			nSrc:     11,
```

Comment:
Why `11`?  I think `10` is correct 🤔

Reply:
Because it reads src to the end even if the src has prefix of old on the suffix.

x/text/transform package passes up to 4096 bytes as src to Transformer. They can't provide more than 4096 bytes for it even when ErrShortSrc occurs, currently(https://github.com/golang/text/blob/master/transform/transform.go#L181).

So we need some buffer to store bytes that have not yet transformed, when we get a fragment of old.

Reply:
Oh, got it :) You are right!